tgui/packages/tgui/interfaces/PoolController.js

```diff
@@ -0,0 +1,82 @@
+import { useBackend } from "../backend";
+import { Button, Flex, LabeledList, Section } from '../components';
+import { Window } from "../layouts";
+
+const TEMPS = {
+  scalding: { label: 'Scalding', color: '#FF0000', icon: 'fa fa-arrow-circle-up', requireEmag: true },
+  warm: { label: 'Warm', color: '#990000', icon: 'fa fa-arrow-circle-up' },
+  normal: { label: 'Normal', color: null, icon: 'fa fa-arrow-circle-right' },
+  cool: { label: 'Cool', color: '#009999', icon: 'fa fa-arrow-circle-down' },
+  frigid: { label: 'Frigid', color: '#00CCCC', icon: 'fa fa-arrow-circle-down', requireEmag: true },
+};
+
+const TempButton = (properties, context) => {
+  const { tempKey, ...buttonProps } = properties;
+  const temp = TEMPS[tempKey];
+  if (!temp) {
+    return null;
+  }
+  const { data, act } = useBackend(context);
+  const { currentTemp } = data;
+  const { label, icon } = temp;
+  const selected = tempKey === currentTemp;
+  const setTemp = () => {
+    act('setTemp', { temp: tempKey });
+  };
+  return (
+    <Button selected={selected} onClick={setTemp} {...buttonProps}>
+      <i className={icon} />
+      {label}
+    </Button>
+  );
+};
+
+export const PoolController = (properties, context) => {
+  const { data } = useBackend(context);
+  const { emagged, currentTemp } = data;
+  const { label: currentLabel, color: currentColor } = TEMPS[currentTemp] || TEMPS.normal;
+
+  const visibleTempKeys = [];
+  for (const [tempKey, { requireEmag }] of Object.entries(TEMPS)) {
+    if (!requireEmag || requireEmag && emagged) {
+      visibleTempKeys.push(tempKey);
+    }
+  }
+
+  return (
+    <Window>
+      <Window.Content>
+        <Section title="Status">
+          <LabeledList>
+            <LabeledList.Item labelColor="yellow" labelClassName="text-bold" label="Current Temperature">
+              <span style={{ color: currentColor }}>
+                {currentLabel}
+              </span>
+            </LabeledList.Item>
+
+            <LabeledList.Item labelColor="yellow" label="Saftey Status">
+              {emagged ? (
+                <span className="text-bold" style={{ color: 'red' }}>
+                  WARNING: OVERRIDDEN
+                </span>
```

Comment:
```suggestion
                <Box color="red">
                  WARNING: OVERRIDDEN
                </Box>
```
Use TGUI elements when you can instead of raw HTML